### PR TITLE
Ignore CustomLayout

### PR DESCRIPTION
**Pull request by A6GibKm**
Fixes: https://github.com/gtk-rs/gtk4-rs/issues/23

### CODE REVIEW

File: gtk4/src/custom_layout.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+unsafe extern "C" fn measure_trampoline<G>(
+    widget: *mut ffi::GtkWidget,
+    orientation: ffi::GtkOrientation,
+    for_size: *mut i32,
+    minimum: *mut i32,
+    natural: *mut i32,
+    minimum_baseline: *mut i32,
+    natural_baseline: *mut i32,
```
Comment (sdroege) [inline gtk4/src/custom_layout.rs:43]: These are all `out` parameters, or maybe even `inout`. You need to write the result to them

File: gtk4/src/custom_layout.rs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                Some(measure_trampoline::<G>),
+                Some(allocate_trampoline::<H>),
+            ))
+        }
+    }
+}
+
+unsafe extern "C" fn request_mode_trampoline<F>(
+    widget: *mut ffi::GtkWidget,
+    f: glib::ffi::gpointer,
```
Comment (sdroege) [inline gtk4/src/custom_layout.rs:27]: All these functions don't take a `user_data` or similar, so you can only really pass `fn()` pointers into them so binding this is rather useless as is. Looks like a bad oversight in the C API, or maybe bindings are instead supposed to properly subclass `gtk::LayoutManager` directly. I guess the latter

**Comment by bilelmoussaoui** [comment]
Per https://github.com/gtk-rs/gtk4-rs/issues/23#issuecomment-766743693, I think we should just add the CustomLayout to the `ignore` list. Do you mind updating the PR to do so? thanks

File: gtk4/Gir.toml [hunk 3]
```diff
@@ -186,6 +185,7 @@ generate = [
 
 # TODO
 ignore = [
+    "Gtk.CustomLayout",
```
Comment (bilelmoussaoui) [inline gtk4/Gir.toml:188]: This needs a comment to that tells why it's ignored, so something like ` # subclass LayoutManager if you need a custom layout`

Comment (bilelmoussaoui) [reply gtk4/Gir.toml:188]: also a rebase

Comment (A6GibKm) [reply gtk4/Gir.toml:189]: Done.

File: gtk4/src/auto/custom_layout.rs [hunk 4]
```diff
@@ -14,12 +14,7 @@ glib::wrapper! {
     }
 }
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/custom_layout.rs:16]: The whole file needs to be removed

**Pull request merged**